storybook/components/DocsContainer/ComponentDocsTemplate/ComponentDocsTemplate.tsx

```diff
@@ -40,6 +41,9 @@ export const ComponentDocsTemplate = ({
       <Content>
         <Title context={context} />
         <Links context={context} />
+        <div className="!mt-8">
```

Comment:
But if it's not in KAIO, wouldn't we get too much margin at the top?

Reply:
Ah you're right, this should be in the notification